apps/site/lib/site_web/controllers/search_controller.ex

```diff
@@ -72,27 +73,29 @@ defmodule SiteWeb.SearchController do
   end
 
   defp do_query(response, conn) do
-    _ = log_error(response)
+    log_error(response)
+
     json(conn, %{error: "bad_response"})
   end
 
-  @spec log_error({:ok, HTTPoison.Response.t()} | {:error, HTTPoison.Error.t() | atom}) ::
-          :ok | {:error, any}
   def log_error({:ok, %HTTPoison.Response{} = response}) do
-    do_log_error(response)
-  end
-
-  def log_error({:error, %HTTPoison.Error{} = response}) do
-    do_log_error(response)
+    response
+    |> Map.from_struct()
+    |> Map.take([:body, :headers, :status_code])
```

Comment:
Might want to omit the `:headers` here. Looking at one of the old `"Received bad response from Algolia"` errors, the response headers are just
```
[{"Server", "nginx"}, {"Date", "Tue, 21 Nov 2023 12:52:06 GMT"}, {"Content-Type", "application/json; charset=UTF-8"}, {"Content-Length", "104"}, {"Connection", "keep-alive"}, {"X-Alg-PT", "1"}, {"Accept-Encoding", "deflate, gzip"}, {"Cache-Control", "no-store"}, {"Access-Control-Allow-Origin", "*"}, {"Timing-Allow-Origin", "*"}, {"X-Content-Type-Options", "nosniff"}, {"Strict-Transport-Security", "max-age=31536000; includeSubDomains; preload"}, {"Content-Disposition", "inline; filename=a.txt"}]
```

Reply:
The issue before was that we were just using `inspect()` to turn the map into a string. Logster will remove any headers that aren't explicitly named in: https://github.com/mbta/dotcom/blob/master/apps/site/config/config.exs#L27